src/integrationTest/java/uk/gov/hmcts/darts/event/service/impl/StandardEventHandlerTest.java

```diff
@@ -269,6 +269,37 @@ void createsAnEventLinkedCaseForEachCaseNumberFromTheDartsEvent() {
             .containsOnly(idFrom(persistedEvents));
     }
 
+    @Test
+    void createsAnEventLinkedCaseForWhenCourtroomDoesntExist() {
```

Comment:
Is this the intended name?
```suggestion
    void createsAnEventLinkedCaseWhenCourtroomDoesntExist() {
```

Reply:
I'll take that